com.ibm.streamsx.topology/info.xml

```diff
@@ -38,8 +38,8 @@ tuple processing using Python.
 See the documentation under [namespace:com.ibm.streamsx.topology.python|com.ibm.streamsx.topology.python] for details.
 
     </description>
-    <version>1.9.1.__dev__</version>
-    <requiredProductVersion>4.2</requiredProductVersion>
+    <version>1.10.0.__dev__</version>
```

Comment:
please don't change info.xml, that unrelated to this PR and the change is incorrect.

info.xml will be changed on the release branch, and 1.10 will continute to be supported on 4.2.